sematic/testing/BUILD

```diff
@@ -0,0 +1,17 @@
+sematic_py_lib(
+    name = "init",
+    srcs = ["__init__.py"],
+    pip_deps = [],
+    deps = [
+        ":mock_funcs",
+    ],
+)
+
+sematic_py_lib(
+    name = "mock_funcs",
+    srcs = ["mock_funcs.py"],
+    pip_deps = [],
+    deps = [
+        "//sematic:calculator",
```

Comment:
Please add an explicit dependency on `pytest`.

Reply:
this module doesn't depend on `pytest`. `unittest.mock` is standard library